release

```diff
@@ -26,7 +26,7 @@ do
   fi
   path="$name"
   log "building $path..."
-  GOOS=$os GOARCH=$arch go build -C .. -ldflags "-w -X github.com/NicksPatties/sweet/cmd/about.version=$version" -o "$builds/$path"
+  GOOS=$os GOARCH=$arch go build -C .. -ldflags "-w -X github.com/NicksPatties/sweet/cmd/about.version=v0.1.0 -X github.com/NicksPatties/sweet/cmd/version.version=$version -X github.com/NicksPatties/sweet/cmd/about.version=$version" -o "$builds/$path"
```

Comment:
This line seems absurd. Do I want to keep doing this each time I want to reference the version of the application in another command?

Maybe I should keep everything in the same package rather than separate everything into their own modules. Is there a way I can have the same directory structure, but have all everything belong to the same module? All of these submodules are nothing by themselves, and are only relevant to the whole sweet application.